Sources/TensorFlow/Initializers.swift

```diff
@@ -39,9 +39,13 @@ extension Tensor {
     repeating repeatedValue: Scalar, shape: TensorShape,
     on device: Device = .default
   ) {
-    self = _Raw.fill(
-      dims: Tensor<Int32>(shape.dimensions.map(Int32.init), on: device),
-      value: Tensor(repeatedValue, on: device))
+    let dims = Tensor<Int32>(shape.dimensions.map(Int32.init), on: device)
+    let value = Tensor(repeatedValue, on: device)
+    #if USING_X10_BACKEND
+      self = _RawXLA.fill(dims: dims, value: value, device: device)
```

Comment:
This one should switch on device.backend. like: https://github.com/tensorflow/swift-apis/blob/master/Sources/TensorFlow/Core/Tensor.swift#L211

Reply:
Fixed the `_RawXLA.fill` implementation instead.